src/Symfony/Component/Form/Tests/ResolvedFormTypeTest.php

```diff
@@ -132,10 +127,6 @@ public function testCreateBuilder()
 
     public function testCreateBuilderWithDataClassOption()
     {
-        if (version_compare(\PHPUnit_Runner_Version::id(), '3.7', '<')) {
-            $this->markTestSkipped('This test requires PHPUnit 3.7.');
-        }
```

Comment:
Can you actually do this cleanup in a separate PR on the older branch containing them ?